techhive-ui/src/components/Internships/Internships.jsx

```diff
@@ -38,7 +39,8 @@ function Internships() {
         const response = await fetch('http://localhost:3000/internships');
         const data = await response.json();
         data.forEach(internship => {
-          internship.isLiked = false; 
+          internship.isLiked = false;
+          internship.isSaved = false; 
```

Comment:
what's the difference between liked and saved? Will they both be used for notifications in the future? 
Will students be able to see both "liked internships" and "saved internships"?